stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -187,63 +187,102 @@ column. In conjunction with [`nonzeros`](@ref) and
 nzrange(S::AbstractSparseMatrixCSC, col::Integer) = getcolptr(S)[col]:(getcolptr(S)[col+1]-1)
 nzrange(S::SparseMatrixCSCView, col::Integer) = nzrange(S.parent, S.indices[2][col])
 
+function isstored(A::AbstractSparseMatrixCSC, i::Integer, j::Integer)
```

Comment:
I feel like there should be an
```
function isstored(A::AbstractArray{<:Any,N}, I::Union{Vararg{Integer,N},CartesianIndex{N}}) where {N}
    @boundscheck checkbounds(A, I...)
    return true
end
```
definition in `Base` that this extends.  e.g. we will probably want to use this in `LinearAlgebra` for various special matrix types.   And there should probably be a definition for `SparseVector`.